### DISCUSSION TITLE
请教个问题

### DISCUSSION THREAD

**Discussion by vihor3** [kaaass/ZerotierFix#98] [Q&A]
从节点列表看，页节点的通信路径变成了中继，并且延迟是-1。然后就是无法访问虚拟内网的设备了，请问这个可能是什么情况？

**Comment by kaaass** [comment]
可能的情况太多了，你可以附上你的网络拓扑、ping 测试的结果等等

**Comment by vihor3** [comment]
![image](https://github.com/kaaass/ZerotierFix/assets/34785243/adcfb495-caa2-4695-8426-13a70ef0db43)
这是listpeers命令的图，下面两个节点一个是PC一个是安卓端，PC用的最新的官方客户端替换planet，安卓用的zerotierfix使用planet。现在是不定时的抽风，突然就打不开了，不管它过个几个小时又自动恢复了，服务器是国内的腾讯云
